test/test-utils.ts

```diff
@@ -17,57 +16,19 @@ export async function getCleanProjectDir(
   name: string
 ): Promise<string> {
   const projectDir = path.resolve(pathPrefix, name);
-  if (fs.existsSync(projectDir)) {
-    await deleteAsync(projectDir, 3);
-  }
-  fs.mkdirSync(projectDir, { recursive: true });
-  return projectDir;
-}
 
-function getMajorNodeVersion(): number {
-  const nodeVersion = process.version.match(/v(\d+)\./);
-  if (nodeVersion && typeof nodeVersion[1] === 'number') {
-    return nodeVersion[1];
-  }
-  return -1;
-}
+  try {
+    await access(projectDir);
+    await rm(projectDir);
+  } catch {}
```

Comment:
if `catch` does not do anything, maybe remove try-catch blocks or just mention "failed to  clean up the project directory"